src/ott/problems/linear/linear_problem.py

```diff
@@ -78,6 +78,16 @@ def is_balanced(self) -> bool:
     """Whether the problem is balanced."""
     return self.tau_a == 1.0 and self.tau_b == 1.0
 
+  @property
+  def is_uniform(self) -> bool:
+    """Test if no weights were passed."""
```

Comment:
Would rephrase the `test`, e.g., `"""True if no marginals were passed."""`.
But am wondering whether the naming of this function is a bit deceptive, since one can pass uniform marginals explicitly and this will not catch it at runtime.
Would maybe elaborate in the docstring for the user.